tools/autodeployment/server.js

```diff
@@ -83,6 +83,24 @@ if (!(DEPLOY_TYPE === 'staging' || DEPLOY_TYPE === 'production')) {
   process.exit(1);
 }
 
+/**
+ * GitHub's webhooks send 3 POST requests with different actions whenever a release event takes place: created, published and released.
+ * To prevent the autodeployment server from processing the 3 requests (and eventually crashing),
+ * we need to only allow the request whose action is 'published'.
+ * More information about release events here:
+ * https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release
+ * The filter combines checking for staging or production build types and also for the right type of action for a release event in production builds
+ * @param {String} name - Repository name (should be 'telescope')
```

Comment:
string vs String for type info. Related, we should think about using doc typing more in backend.

Reply:
Should I also change the types in lines 88 and 89 [here](https://github.com/Seneca-CDOT/telescope/blob/f35311a09e179223ea92926493c91aac9897c7d3/tools/autodeployment/server.js#L88)?

Reply:
Yeah, do it throughout.  Built-in types should be lowercase.